packages/sdks/web-component/src/lib/helpers/helpers.ts

```diff
@@ -716,9 +716,24 @@ export const transformStepStateForCustomScreen = (
     sanitizedState.action = state.action;
   }
 
+  if (state.screenState?.componentsConfig?.thirdPartyAppApproveScopes?.data) {
+    sanitizedState.inboundAppApproveScopes =
+      state.screenState.componentsConfig.thirdPartyAppApproveScopes.data;
+  }
+
   return sanitizedState;
 };
 
+export const transformScreenInputs = (inputs: Record<string, any>) => {
+  const res = { ...inputs };
+
+  if (inputs.inboundAppApproveScopes) {
+    res.thirdPartyAppApproveScopes = inputs.inboundAppApproveScopes;
```

Comment:
this is for BC, no? worth a comment on that

also what if `inboundAppApproveScopes` is empty? can't we have `res.thirdPartyAppApproveScopes = inputs.inboundAppApproveScopes;`?

Reply:
If it's empty we do not need to do anything
This is for custom screens
we want to expose an API that includes the updated names so we are mapping it to the "old" names

Reply:
ok